src/interface_r/DESCRIPTION

```diff
@@ -0,0 +1,28 @@
+Package: h2o4gpu
+Type: Package
+Title: h2o4gpu in R
+Version: 0.0.1.9999
+Authors@R: c(
+  person("Yuan", "Tang", role = c("aut", "cre"), email = "terrytangyuan@gmail.com"),
+  person("H2O.AI", role = c("cph", "fnd")))
+Maintainer: Yuan Tang <terrytangyuan@gmail.com>
```

Comment:
For the maintainer can you make it multiple people? I forget if you can. Would be good if its me, you, and @ledell.